pyvista/utilities/transformations.py

```diff
@@ -0,0 +1,39 @@
+"""Transformations leveraging transforms3d library."""
+import numpy as np
+import transforms3d as tf3d
+
+
+def axis_angle_rotation(axis, angle, point=None, deg=True):
+    """Return a 4x4 matrix for rotation about an axis by given angle, optionally about a given point."""
+    if deg:
+        angle *= np.pi / 180
+    return tf3d.axangles.axangle2aff(axis, angle, point=point)
+
+
+def reflection(normal, point=None):
+    """Return a 4x4 matrix for reflection across a normal about a point."""
+    return tf3d.reflections.rfnorm2aff(normal, point=point)
+
+
+def apply_transformation_to_points(transformation, points, inplace=False):
+    """Apply a given transformation matrix (3x3 or 4x4) to a set of points."""
+    if transformation.shape not in ((3, 3), (4, 4)):
+        raise RuntimeError('`transformation` must be of shape (3, 3) or (4, 4)')
```

Comment:
Should be `ValueError`.  `RuntimeError` is really a catch-all.  See [RuntimeError](https://docs.python.org/3/library/exceptions.html#RuntimeError)
```suggestion
        raise ValueError('`transformation` must be of shape (3, 3) or (4, 4)')
```